modules/fedimint-ln-server/src/lib.rs

```diff
@@ -1035,6 +1035,7 @@ mod fedimint_migration_tests {
                 .expect("Could not parse URL to generate GatewayClientConfig API endpoint"),
             route_hints: vec![],
             valid_until: SystemTime::now(),
+            fees_proportional_millionths: 0,
```

Comment:
Why is this 0?

Reply:
this test suite doesn't assert on the actual values used. Rather it only asserts on the fields of the struct. Using `0` as a random placeholder value on the new field `fees_proportional_millionths` should be harmless, and a good verification of the test assertions imo

Reply:
Ah I missed that this was the migration test. Yeah that's fine.